runners/spark/spark_runner.gradle

```diff
@@ -221,20 +221,17 @@ hadoopVersions.each { kv ->
   }
 }
 
-def validatesRunnerBatch = tasks.register("validatesRunnerBatch", Test) {
-  group = "Verification"
```

Comment:
Please keep the group, looks like this got missing

Reply:
Fixed